examples/apps/spaces/src/fluid-object/spacesToolbar.tsx

```diff
@@ -151,7 +152,7 @@ export const SpacesToolbar: React.FC<ISpacesToolbarProps> =
                     <SpacesToolbarAddTemplatePicker
                         key="templates"
                         templates={props.templates}
-                        applyTemplate={props.applyTemplate}
+                        applyTemplate={props.applyTemplate.bind(this)}
```

Comment:
This looks wrong - also `applyTemplate` should be fine without a bind since it's already an arrow function in `Spaces`

Reply:
Is this only safe because we know the only implementer of `ISpacesToolbarProps` is ours? Is unbound-method going to be super-noisy for our React projects then? Because it seems like there's no way for it to ignore this case. Maybe there's a React-specific rule we should be using instead?

Reply:
Hmmm, maybe this is a typings issue. `props.applyTemplate` is an any within the JSX -- I don't understand why.

Reply:
Maybe we shouldn't use React.FC at all: <https://github.com/facebook/create-react-app/pull/8177>

Reply:
I'd say type the props with arrow functions, which generally satisfies the lint rule.

It's not a matter of being safe here though, calling bind(this) at this location would bind to the incorrect "this".  The binding needs to be done from the context of the implementer of applyTemplate.

For React.FC we generally are using it with an explicitly typed props param which should prevent the implicit children problem I believe.

Reply:
> It's not a matter of being safe here though, calling bind(this) at this location would bind to the incorrect "this". The binding needs to be done from the context of the implementer of applyTemplate.

Thanks, I think I get it now. Typing with arrow props is an elegant solution.